near-contract-standards/src/non_fungible_token/metadata.rs

```diff
@@ -5,6 +5,8 @@ use near_sdk::serde::{Deserialize, Serialize};
 
 /// This spec can be treated like a version of the standard.
 pub const NFT_METADATA_SPEC: &str = "nft-1.0.0";
+/// This name of the NFT standard we're using
```

Comment:
nit: public comment, so better to be grammatically correct about
```suggestion
/// This is the name of the NFT standard we're using
```